portal-web/test/functional/com/liferay/portalweb/tests/enduser/adaptivemedia/AdaptiveMedia.testcase

```diff
@@ -237,6 +237,55 @@
 		<execute macro="AdaptiveMedia#viewNoImageResolution" />
 	</command>
 
+	<command name="AddImageResolutionWithNoDimensions" priority="4">
+		<execute macro="ProductMenu#gotoPortlet">
+			<var name="category" value="Configuration" />
+			<var name="panel" value="Control Panel" />
+			<var name="portlet" value="Adaptive Media" />
+		</execute>
+
+		<execute macro="AdaptiveMedia#addImageResolution">
+			<var name="imageName" value="testResolution" />
+			<var name="maxHeight" value="" />
+			<var name="maxWidth" value="" />
+			<var name="noDimensions" value="true" />
+		</execute>
```

Comment:
@acbbennett please assert that the image resolution is actually added (think of the MB test you wrote for the training assessment).